scrapypuppeteer/response.py

```diff
@@ -98,26 +101,57 @@ class PuppeteerScreenshotResponse(PuppeteerResponse):
     Screenshot is available via self.screenshot as base64 encoded string.
     """
 
-    attributes: Tuple[str, ...] = PuppeteerResponse.attributes + (
-        'screenshot',
-    )
+    attributes: Tuple[str, ...] = PuppeteerResponse.attributes + ("screenshot",)
 
     def __init__(self, url, puppeteer_request, context_id, page_id, **kwargs):
-        self.screenshot = kwargs.pop('screenshot')
+        self.screenshot = kwargs.pop("screenshot")
         super().__init__(url, puppeteer_request, context_id, page_id, **kwargs)
 
 
 class PuppeteerJsonResponse(PuppeteerResponse):
     """
-    Response for CustomJsAction and RecaptchaSolver.
+    Response for CustomJsAction.
     Result is available via self.data object.
     """
 
-    attributes: Tuple[str, ...] = PuppeteerResponse.attributes + (
-        'data',
-    )
+    attributes: Tuple[str, ...] = PuppeteerResponse.attributes + ("data",)
 
     def __init__(self, url, puppeteer_request, context_id, page_id, data, **kwargs):
-        kwargs['headers'] = {'Content-Type': 'application/json'}
+        kwargs["headers"] = {"Content-Type": "application/json"}
         self.data = data
         super().__init__(url, puppeteer_request, context_id, page_id, **kwargs)
+
+
+class PuppeteerRecaptchaSolverResponse(PuppeteerJsonResponse, PuppeteerHtmlResponse):
+    """
+    Response for RecaptchaSolver.
+    Result is available via self.recaptcha_data and self.data["recaptcha_data"]
+    (deprecated, to be deleted in next versions) object.
+    """
+
+    attributes: Tuple[str, ...] = tuple(
+        set(PuppeteerHtmlResponse.attributes + PuppeteerJsonResponse.attributes)
+    ) + ("recaptcha_data",)
```

Comment:
Sorry for this mess. We are not able to concatenate these two tuples easily as we have inheritance from 2 responses for now.